app/models/submission_collector.rb

```diff
@@ -212,6 +227,8 @@ def manually_collect_submission(grouping, rev_num, async=true)
     grouping.save
 
     new_submission = Submission.create_by_revision_number(grouping, rev_num)
+    new_submission = apply_penalty_or_add_grace_credits(grouping, 
```

Comment:
Useless assignment to variable - `new_submission`.<br>Trailing whitespace detected.